frontend/src/components/ShootDetails/GShootKubeconfig.vue

```diff
@@ -45,24 +45,61 @@ SPDX-License-Identifier: Apache-2.0
         </span>
       </template>
     </g-list-item-content>
+    <g-list-item-content v-if="isAdminKubeconfigType">
+      Admin Kubeconfig
+      <div class="text-body-2 d-flex">
+        <span>
+          Request a kubeconfig valid for
+        </span>
+        <g-popover
+          toolbar-title="Configure Admin Kubeconfig Lifetime"
+          placement="bottom"
+        >
+          <template #activator="{ props }">
+            <v-chip
+              label
+              size="x-small"
+              color="primary"
+              variant="outlined"
+              class="ml-2 pointer"
```

Comment:
do not use margin inside a popover. Add the margin to the text before the activator.